lib/iris/tests/test_cdm.py

```diff
@@ -692,7 +692,7 @@ def test_cube_iteration(self):
                 pass
 
     def test_not_iterable(self):
-        self.assertFalse(isinstance(self.t, collections.Iterable))
+        self.assertFalse(isinstance(self.t, collections.abc.Iterable))
```

Comment:
```
  /tmp/cirrus-ci-build/lib/iris/tests/test_cdm.py:695: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
```